tests/conftest.py

```diff
@@ -48,8 +48,7 @@ def pytest_logger_config(logger_config):
     logger_config.set_formatter_class(OCSLogFormatter)
 
 
-ep_time = int(time.time())
-config.RUN['run_id'] = ep_time
+ep_time = config.RUN['run_id']
```

Comment:
This is not used anywhere else so we can probably just use this in log_dir on line 52:
`logs_{config.RUN['run_id']}`

Reply:
Fixed by 6e0d644.